appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -460,89 +437,69 @@
   @Description("Name of Save as ... menuitem")
   String saveAsMenuItem();
 
-  @DefaultMessage("Checkpoint")
+  @DefaultMessage("Save a checkpoint")
   @Description("Name of Checkpoint menuitem")
   String checkpointMenuItem();
 
-  @DefaultMessage("Import project (.aia) from a repository ...")
+  @DefaultMessage("Import project (.aia) from a template ...")
   @Description("Name of Import Template menuitem")
   String importTemplateButton();
 
   @DefaultMessage("Export selected project (.aia) to my computer")
   @Description("Name of Export Project menuitem")
   String exportProjectMenuItem();
 
+  @DefaultMessage("Export this project (.aia) to my computer")
+  @Description("Name of Export This Project menuitem")
+  String exportThisProjectMenuItem();
+
   @DefaultMessage("Export all projects")
   @Description("Name of Export all Project menuitem")
   String exportAllProjectsMenuItem();
 
-  @DefaultMessage("Export keystore")
+  @DefaultMessage("Export app signing keystore")
   @Description("Label of the button for export keystore")
   String downloadKeystoreMenuItem();
 
-  @DefaultMessage("Import keystore")
+  @DefaultMessage("Import app signing keystore")
   @Description("Label of the button for import keystore")
   String uploadKeystoreMenuItem();
 
-  @DefaultMessage("Delete keystore")
+  @DefaultMessage("Delete app signing keystore")
   @Description("Label of the button for delete keystore")
   String deleteKeystoreMenuItem();
 
-  //Connect
-  @DefaultMessage("Connect")
+  //Live-test
+  @DefaultMessage("Test")
   @Description("Label of the button leading to Connect related cascade items")
   String connectTabName();
 
-  @DefaultMessage("AI Companion")
+  @DefaultMessage("Companion App over WiFi")
   @Description("Message providing details about starting the wireless connection.")
-  String AICompanionMenuItem();
+  String aiCompanionMenuItem();
```

Comment:
@ajcolter You can make this change, but then you should update all of the translation properties files as well (and we have to update all of the translations in Pootle!).